app/segments/drill/[id]/assignment/index.js

```diff
@@ -78,14 +98,14 @@
     const selectedUsers = Object.entries(checkedItems)
       .filter(([, value]) => value)
       .map((value) => value[0]);
-    const time = new Date().getTime();
+    const time = getTimezoneOffsetTime(new Date().getTime());
```

Comment:
Let's standardize all time to UTC until you actually display them. This way assignments that are assigned in different time zones can still work.

Reply:
Also if we're gonna do 1 drill per day, let's round this to the current day (ofc still in timestamp format)